core/trino-main/src/main/java/io/trino/operator/scalar/json/JsonInputFunctions.java

```diff
@@ -56,7 +59,13 @@ public final class JsonInputFunctions
     public static final String VARBINARY_UTF16_TO_JSON = "$varbinary_utf16_to_json";
     public static final String VARBINARY_UTF32_TO_JSON = "$varbinary_utf32_to_json";
 
-    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
It's called in a couple of places (i.e. tests) and in most of them it's harmless but in the engine code it should be banned because it operates on both user input and these instances live long.

Reply:
makes sense. so how do we prevent new not harmless direct uses of ObjectMapper?